packages/muelu/src/MueCentral/MueLu_FactoryManager_def.hpp

```diff
@@ -194,14 +195,17 @@ namespace MueLu {
       if (varName == "Graph")                           return MUELU_KOKKOS_FACTORY(varName, CoalesceDropFactory, CoalesceDropFactory_kokkos);
       if (varName == "UnAmalgamationInfo")              return MUELU_KOKKOS_FACTORY(varName, AmalgamationFactory, AmalgamationFactory_kokkos);
       if (varName == "Aggregates")                      return MUELU_KOKKOS_FACTORY(varName, UncoupledAggregationFactory, UncoupledAggregationFactory_kokkos);
-      if (varName == "AggregateQualities")       return SetAndReturnDefaultFactory(varName, rcp(new AggregateQualityEstimateFactory()));
+      if (varName == "AggregateQualities")       		return SetAndReturnDefaultFactory(varName, rcp(new AggregateQualityEstimateFactory()));
```

Comment:
@maxfirmbach Please either revert this change in indentation or align it with the surrounding lines.